### PR TITLE
Use custom widgets

### LINKED ISSUE #91: extra slider on Mac
Running on Mac there is an extra slider that shows up on top of the start date:

image

### LINKED ISSUE #131: Replace TextWindow
Can we replace TestWindow by a QTextBrowser?

### LINKED ISSUE #133: use custom Qt widgets in designer
It would be nice if ...

... the custom widgets defined here (such as [`DateTimeRangeSlider`](https://github.com/BCDA-APS/gemviz/blob/main/gemviz/date_time_range_slider.py) or [`RangeSlider`](https://github.com/BCDA-APS/gemviz/blob/main/gemviz/range_slider.py)) by Python code could be used in other widgets in the Qt Designer.

### PR DESCRIPTION
**Pull request by rodolakis**
- close #91 
- close #133 
- close #131

### CODE REVIEW

**Comment by prjemian** [comment]
When resizing the LICENSE window, the text area did not adjust to the new geometry.
image

File: gemviz/resources/licensedialog.ui [hunk 1]
```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ui version="4.0">
+ <class>Dialog</class>
+ <widget class="QDialog" name="Dialog">
```
Comment (prjemian) [inline gemviz/resources/licensedialog.ui:4]: Apply vertical layout (in designer) to this widget.

File: gemviz/resources/licensedialog.ui [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    <x>0</x>
+    <y>0</y>
+    <width>530</width>
+    <height>560</height>
+   </rect>
+  </property>
+  <property name="windowTitle">
+   <string>Dialog</string>
+  </property>
+  <widget class="QWidget" name="verticalLayoutWidget">
```
Comment (prjemian) [inline gemviz/resources/licensedialog.ui:16]: Remove this intermediate QWidget.

File: gemviz/resources/licensedialog.ui [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+   <property name="geometry">
+    <rect>
+     <x>0</x>
+     <y>0</y>
+     <width>530</width>
+     <height>560</height>
+    </rect>
+   </property>
+   <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
```
Comment (prjemian) [inline gemviz/resources/licensedialog.ui:25]: First, break the layout here.  That will remove this QVBoxLayout and free up the QTextBrowser to be managed by the window's layout.

**Comment by rodolakis** [comment]
Done.

**Review by prjemian — APPROVED** [review]
LGTM!

**Pull request merged**